### ISSUE TITLE
[Feature] - Plot request by address

### ISSUE THREAD

**Issue by XylotrupesGideon** [AlpsBTE/Plot-System#71]
In the new plot system it would be great t give builders the option to request buildings by address. This way the builders could use the plot system to build a specific building.

Important for that would be:
1. First time builders should not request a building this way it should be connected to the builder score in soem way maybe as a reward for buildign at least 2 buildings?
2. the request must be anonymous so the person creating the plot on the build team side can not conenct it to a specific builder (this is very important as many people would request their home address)

The buidler would choose a country to build in and then have the option to request a plot by address. They would send of thier request and get a message saying: "Thank you for the requuest, Our team will prepare the plot as soon as possible and notify you when it is ready. Why not build a different plot while you are waiting."For the person an empty plot wuld be temporarily generated with the ID X.

On the BT side a "plotter" will be notified that someone requested the plot with the given address with the ID X with a link to google maps so they can find out the coordinates.
They would then prepare the plot for the person with soem command like /create plot ABC so the plot gets sorted to the right ID.
Alternatively if there is a problem (e.g. the address does not exsist or the map is corrupted or it is a joke or something) the plotter can deny the request with an information message.

If the plot is prepared the builder gets a message saying "Your Requested plot has been generated use /project X to start building. Have fun"

**Comment by Georg-code** [comment]
I would like to give my opinion on this briefly.
1) One of the main ideas of the plot system, if I understand it correctly, is to bundle progress.  This would go against that.
2) If a builder has already built 2 plots, he can apply and then build on the server of the build team. He has then already fulfilled the prerequisites for this.
3) As I understand it, it is very time-consuming for the "plotters" to prepare the plots atm. I think that with such a system it would make everything much more complicated.
I love the idea, but I think it is out of scope.

**Comment by NApartas** [comment]
I agree with Hohi. With the plot system it does make no sense. Being an interesting topic though.

**Comment by XylotrupesGideon** [comment]
Yes I guess that is the main purpose. maybe it is a feture for the future in case the system evolves
